lldb/source/Target/Platform.cpp

```diff
@@ -1197,6 +1197,32 @@ Status Platform::PutFile(const FileSpec &source, const FileSpec &destination,
   if (!source_file)
     return Status(source_file.takeError());
   Status error;
+
+  bool requires_upload = true;
+  uint64_t dest_md5_low, dest_md5_high;
+  bool success = CalculateMD5(destination, dest_md5_low, dest_md5_high);
+  if (!success) {
+    LLDB_LOGF(log, "[PutFile] couldn't get md5 sum of destination");
+  } else {
+    auto local_md5 = llvm::sys::fs::md5_contents(source.GetPath());
```

Comment:
It's not obvious from the right hand side what the return value is. Case in point, it returns an `ErrorOr<MD5::MD5Result>` and if I hadn't looked it up I wouldn't have noticed we're not actually logging the error.